### PR TITLE
Improve help + flags, update README

### PR DESCRIPTION
**Pull request by prashantv**
Flags now pass through unknown flags (like `--alloc_objects`) to `go tool pprof`, and are broken into multiple sections.

Restructure the README into:
- Usage
- Integration
- Installation
- Development

Can see the rendered output here:
https://github.com/uber/go-torch/blob/readme/README.md

Focus examples on more common usage, and remove less frequently used examples like outputting an SVG to stdout or outputting the raw flamegraph input.

### CODE REVIEW

**Comment by sandlerben** [comment]
This is awesome. It's really cool to see the project getting better and
better.

File: README.md [hunk 1]
```diff
@@ -14,70 +14,104 @@ synthesizes them into a flame graph. Uses Go's built in [pprof][] library.
...
-INFO[0000] Profiling ...
-INFO[0000] flame graph has been created as torch.svg
+$ go-torch --alloc_objects main.test mem.prof
+INFO[19:00:29] Run pprof command: go tool pprof -raw -seconds 30 --alloc_objects main.test prof.mem
+INFO[19:00:29] Writing svg to torch.svg
 ```
 
+## Integrating With Your Application
+
+To profile profiling endpoints in your application, follow the official
```
Comment (sectioneight) [inline README.md]: Somewhat awkward wording: "To profile profiling..."

File: torchlog/torchlog.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func init() {
 func getPrefix(level string, color *color.Color) string {
 	currentTime := time.Now().Format("15:04:05")
 	toColoredString := color.SprintFunc()
-	return toColoredString(fmt.Sprintf("%s[%s]", level, currentTime))
+	return toColoredString(fmt.Sprintf("%s[%s] ", level, currentTime))
```
Comment (sectioneight) [inline torchlog/torchlog.go]: Is this related to this change?

**Comment by sectioneight** [comment]
Looks good to me!

File: README.md (hunk 1 above)
Comment (prashantv) [inline README.md]: Good catch, meant "To add profiling", updated.

File: torchlog/torchlog.go (hunk 2 above)
Comment (prashantv) [inline torchlog/torchlog.go]: Minor formatting change to make the output a little nicer. The previous output looked like:
```
INFO[19:00:29]Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
INFO[19:00:29]Writing svg to torch.svg
```

I thought adding a space would make the output a little more readable:
```
INFO[19:00:29] Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
INFO[19:00:29] Writing svg to torch.svg
```

Caught this while updating the examples in `README`

Comment (sectioneight) [inline torchlog/torchlog.go]: Yup, agree it's better. Wasn't sure whether you meant to include it here or
another diff. This is good though.
